src/DearImGui/Raw/IO.hs

```diff
@@ -120,14 +120,11 @@ setUserData ptr = liftIO do
 
 {- TODO:
 
-bool   WantCaptureMouse;         // Set when Dear ImGui will use mouse inputs, in this case do not dispatch them to your main game/application (either way, always pass on mouse inputs to imgui). (e.g. unclicked mouse is hovering over an imgui window, widget is active, mouse was clicked over an imgui window, etc.).
-bool   WantCaptureKeyboard;      // Set when Dear ImGui will use keyboard inputs, in this case do not dispatch them to your main game/application (either way, always pass keyboard inputs to imgui). (e.g. InputText active, or an imgui window is focused and navigation is enabled, etc.).
```

Comment:
While adding `framerate`, I noticed these were already available in `DearImGui.Raw`